docs/3_how_to_filter_articles.md

```diff
@@ -75,15 +75,15 @@ for article in crawler.crawl(max_articles=5, url_filter=regex_filter("advertisem
 Often it's useful to select certain criteria rather than filtering them.
 To do so use the `inverse` operator from `fundus.scraping.filter.py`.
 
-Let's crawl a bunch of articles with URLs including the string `politic`.
+Let's crawl a bunch of articles with URLs including the string `politics`.
```

Comment:
That's not a typo but on purpose. `politic` -> `politics`; `politic` -> `political` etc.

Reply:
Sorry for the misconception, Alright, will be changed.